packages/kbn-esql-validation-autocomplete/src/validation/validation.test.ts

```diff
@@ -3048,7 +3052,9 @@ describe('validation logic', () => {
 
       describe('mv_zip', () => {
         testErrorsAndWarnings('row var = mv_zip("a", "a", "a")', []);
+        testErrorsAndWarnings('row var = mv_zip("a", "a")', []);
```

Comment:
The new definition for `mv_zip` fixes a bug since until now assumed you had to give 3 arguments, but 2 is actually fine.